omnisette/src/store_services_core.rs

```diff
@@ -66,18 +66,18 @@ pub struct StoreServicesCoreADIProxy<'lt> {
 }
 
 impl StoreServicesCoreADIProxy<'_> {
```

Comment:
Just a small note, back when I wrote omnisette I wasn't aware that there was a cleanup function to call in CoreADI. Maybe adding a drop implementation for that struct would be a good idea for production (it won't make CoreADI stop leaking but it will at least leak less).

Reply:
I don't use CoreADI here, so :shrug: different pr?